autoload/test/gleam/gleamtest.vim

```diff
@@ -0,0 +1,34 @@
+if !exists('g:test#gleam#gleamtest#file_pattern')
+  let g:test#gleam#gleamtest#file_pattern = '\v\.gleam$'
+endif
+
+function! test#gleam#gleamtest#test_file(file) abort
+  return a:file =~# g:test#gleam#gleamtest#file_pattern
+endfunction
+
+function! test#gleam#gleamtest#build_position(type, position) abort
+  if a:type ==# 'nearest'
+    return ['test', a:position['file']]
+  elseif a:type ==# 'file'
+    return ['test', a:position['file']]
+  else
+    return []
+  endif
+endfunction
+
+function! test#gleam#gleamtest#build_args(args)
+  if empty(filter(copy(a:args), 'test#base#file_exists(v:val)'))
+    call add(a:args, 'build test')
```

Comment:
Looking at the [gleam CLI docs](https://gleam.run/writing-gleam/command-line-reference/) are tests invoked with `gleam test`?

```suggestion
    call add(a:args, 'test')
```

Reply:
Yes, I guess there is no need to call build too.